src/lib/utils.ts

```diff
@@ -5,3 +5,23 @@ import { twMerge } from 'tailwind-merge';
 export function cn(...inputs: ClassValue[]) {
   return twMerge(clsx(inputs));
 }
+
+export function getRandomElements<T>(arr: T[], count: number): T[] {
+  const shuffled = arr.slice();
+  let i = arr.length;
+  let temp: T, index: number;
+
+  // Fisher-Yates shuffle algorithm
+  while (i--) {
+    index = Math.floor((i + 1) * Math.random());
+    temp = shuffled[i];
+    shuffled[i] = shuffled[index];
+    shuffled[index] = temp;
+  }
+
```

Comment:
seems quite complicated?
What about something like this:
```
return arr.sort(() => 0.5 - Math.random()).slice(0, count))
```

Reply:
I took the Fisher-Yates because it seems to have the lowest complexity. On the other hand performance doesn't matter in this case, so I agree having simpler way is better 👍